tools/linter/README.md

```diff
@@ -0,0 +1,33 @@
+# api.yaml linter
+
+This linter lints api.yaml against the Discovery Docs.
+
+Because api.yaml is handwritten, it will inevitably veer away from the Discovery docs
+(the source of truth) or just be inputted incorrectly.
+
+## Running the linter
+The linter is powered by RSpec.
+
+```
+  rspec tools/linter/run.rb
+```
+
+## Adding new tests
```

Comment:
We're using `tests` to describe what are really checks; it's just an implementation detail that they are tests. Do you think it would be more clear if we called them that here + in `run.rb`?